tests/unit/amo/components/TestSearchResult.js

```diff
@@ -64,35 +64,44 @@ describe(__filename, () => {
   it('renders the heading', () => {
     const root = render();
 
-    expect(root.find('.SearchResult-link').children()).toIncludeText(
+    expect(root.find('.SearchResult-title').children()).toIncludeText(
       'A search result',
     );
   });
 
-  it('links the heading to the detail page', () => {
+  it('links the card to the detail page', () => {
     const slug = 'some-addon-slug';
     const addon = createInternalAddon({ ...fakeAddon, slug });
 
     const root = render({ addon });
 
-    expect(root.find('.SearchResult-link')).toHaveProp('to', getAddonURL(slug));
+    expect(root.find('.SearchResult-wrapper')).toHaveProp(
+      'to',
+      getAddonURL(slug),
+    );
   });
 
-  it('stops propagation when clicking on the add-on name', () => {
+  it('links the card to do nothing when addon is null', () => {
+    const root = render({ addon: null });
+
+    expect(root.find('.SearchResult-wrapper')).toHaveProp('href', '#');
```

Comment:
This is good, but I think, because we have logic that creates either a `to` or an `href` attribute, we would also be checking that there is no `to` attribute in this test.